### ISSUE TITLE
Peer info panel

### ISSUE THREAD

**Issue by ckerr** [transmission/transmission#2537]
### Discussed in https://github.com/transmission/transmission/discussions/2534

<div type='discussions-op-text'>

<sup>Originally posted by **GScottElshaw** January 29, 2022</sup>
What if the peer info panel **only** showed peers that are connected and downloading or being seeded to? What if all other connections were never reported to the peers window, and instead the information about connections were placed at the top of the window such as this? The obvious advantages are screen real estate and a less traffic-intensive, and ultimately, a more meaningful environment. 
![Screenshot 2022-01-29 at 9 09 54 PM](https://user-images.githubusercontent.com/69029666/151662612-f482deb9-1302-44e7-9661-b305f9625277.png)

</div>

**Comment by mangosplat** [comment]
I would like to continue to see the IP addresses of all connected peers. -1

**Comment by GScottElshaw** [comment]
Is there a particular reason?

**Comment by Pentaphon** [comment]
I think its best to just do what all other clients do instead of obfuscating information for the sake of simplicity. We need Transmission to do as much as the other clients out there, and to do it better, not have it do less.

**Comment by GScottElshaw** [comment]
I disagree that it obfuscates information, i think it has the opposite affect, all those 0.0 and 100's are a meaningless firehose of data that belong in a console like Transmission's Message Log. They are would-be potential peers, but not peers per se.

Just in case i'm being perceived as the 'less-is-more guy', i'm on a Mac, and if Transmission is somehow not behaving when downloading a torrent, I have to use uTorrent which is now web-based and really is so minimalistic that it feels like everything is a roll of the dice. I don't think that is a useful model for torrent clients.

**Comment by Pentaphon** [comment]
> I disagree that it obfuscates information

But it does. It's still information. It's just that YOU think it's meaningless, so you want to obfuscate it. It's really that simple. 

I think we should always meet the standards of BitTorrent clients today. Since they all show that information, Transmission should show it as well.

**Comment by GScottElshaw** [comment]
Okey-doke.

**Comment by Coding4Coffee** [comment]
My thoughts are the inspector window is to do just that, to inspect and therefore should have all the data. Typically, you aren't looking at the peer window unless something is wrong.

**Comment by ckerr** [comment]
The Qt and web clients default to having a pretty concise list of information in the Trackers tab to avoid being overwhelming, but there is a "[x] show more info" checkbox underneath to toggle on a more verbose display. Maybe something like this could be used to control what peer information is listed too

**Comment by Pentaphon** [comment]
> Maybe something like this could be used to control what peer information is listed too

Works for me.

**Comment by pmarsden** [comment]
All connected peers should be shown, thats what the display is for. If Im looking at that window, I want all the information, not some of it.  As far as 100% peers are concerned, my experience is they only show up if you dont have 100% yourself, so are seeders you will potentially be downloading from. Peers at 0% are sometimes problem peers, who stay on 0%, even though they are [constantly] downloading. I have had to ban a number of these over the last 3 years.

**Comment by ckerr** [comment]
Well this feature request's been here for four years now and there's no consensus on it. I'm going to mark it as "declined" and move on.
